src/users/users.service.ts

```diff
@@ -13,18 +13,25 @@ export class UsersService {
     @InjectModel(Users)
     private readonly usersRepository: Repository<Users>,
     @InjectModel(UsersGroups)
-    private readonly usersGroupsRepository: Repository<UsersGroups>, // @Inject('SequelizeInstance') // private readonly sequelizeInstance,
+    private readonly usersGroupsRepository: Repository<UsersGroups>,
   ) {}
 
   getAll() {
     return this.usersRepository.findAll();
   }
 
-  getById(id: ID) {
-    return this.usersRepository.findByPk(id);
+  async getById(id: ID) {
+    const user = await this.usersRepository.findByPk(id);
+    if (!user) throw new HttpException('This login already uses', HttpStatus.BAD_REQUEST);
+
```

Comment:
it's a bad idea to throw Http exceptions in service layers...

Reply:
use controllers for that

Reply:
fixed

Reply:
fixed